nixos/tests/ayatana-indicators.nix

```diff
@@ -39,6 +40,7 @@ in {
     # Setup needed by some indicators
 
     services.accounts-daemon.enable = true; # messages
+    # services.geoclue2.enable = true; # display, maybe not a good idea to enable this for reproducibility?
```

Comment:
Do you remember why this is then not reproducible?

Reply:
I haven't tested this, I just didn't think that tests should have any geolocation interactions happening inside of them. AI display can use `xsct` to change the screen temperature based on geoclue information, not sure if that could i.e. mess up any of the OCR.

Reply:
If the screen temperature gets to red, then the lower contrast could mess with things. I would say we just drop this comment to not confuse us in the future.

Reply:
Dropped.